src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -696,3 +697,46 @@ unittest
     auto d = Deserializer.deserialize!(S2)(buffer);
     test(deepEquals(*d.ptr, s));
 }
+
+/******************************************************************************
+
+    Const arrays of arrays
+
+******************************************************************************/
+
+struct ConstS
+{
+    Const!(int) n = 3;
+    Const!(char[][]) a = ["Hello", "World"];
+    Const!(char[])[2][3] b = [
+        ["Die", "Katze"], ["tritt", "die"], ["Treppe", "krumm."]
+    ];
+}
+
+unittest
+{
+    void[] buffer;
+
+    ConstS s;
+
+    Serializer.serialize(s, buffer);
+    auto cont_S = Deserializer.deserialize!(ConstS)(buffer);
```

Comment:
Sorry, I didn't read your comment very well :blush: -- let me check if there was a complication with the original plan.